pipeline/errors/error_redirect_test.go

```diff
@@ -42,12 +42,30 @@ func TestErrorRedirect(t *testing.T) {
 				},
 			},
 			{
-				d:          "should redirect with 301",
+				d:          "should redirect with 302 - relative",
 				givenError: &herodot.ErrNotFound,
-				config:     `{"to":"http://test/test","code":301}`,
```

Comment:
Can you please add new test cases instead of replacing the old ones?

Reply:
Hi @zepatrik , 

The new test cases were added - I think the diff looks a little messy because I put 302-relative with 302-absolute, and moved the 301 below  it, whereas previously it was 302-absolute and 301-absolute.

In total, there are 4 test cases there, the original being the absolute ones, with only the description being added in.

I fixed an issue in the previous commits where I accidentally removed the absolute test for 301, it should be fine now.

Reply:
NIce thanks